helm/cluster-aws/values.schema.json

```
@@ -175,32 +242,43 @@
                     "$ref": "#/$defs/machinePool"
                 }
             },
+            "title": "Default node pool",
             "type": "object"
         },
         "flatcarAWSAccount": {
+            "description": "AWS account ID owning the Flatcar Container Linux AMI.",
+            "title": "AWS account owning Flatcar image",
```

Is it better to use "image" than "AMI"?

```suggestion
            "title": "AWS account owning Flatcar AMI",
```

---

We avoid abbreviations where possible